inst/extdata/fd_plots.csv

```diff
@@ -1,17 +1,17 @@
-Replicate,Plot,Longitude,Latitude,Disturbance_severity
-A,1,-100,47,0
-A,2,-100,47,45
-A,3,-100,47,65
-A,4,-100,47,85
-B,1,-100,46,0
-B,2,-100,46,45
-B,3,-100,46,65
-B,4,-100,46,85
-C,1,-101,46,0
-C,2,-101,46,45
-C,3,-101,46,65
-C,4,-101,46,85
-D,1,-101,47,0
-D,2,-101,47,45
-D,3,-101,47,65
-D,4,-101,47,85
+﻿Replicate,Plot,Latitude,Longitude,Plot_area_m2
+A,1,45.563404,-84.728465,5000
+A,2,45.56418,-84.727971,5000
+A,3,45.562995,-84.727099,5000
+A,4,45.563701,-84.726382,5000
+B,1,45.556685,-84.72685,5000
+B,2,45.555822,-84.726783,5000
+B,3,45.556236,-84.725704,5000
+B,4,45.556248,-84.724383,5000
+C,1,45.562496,-84.707799,5000
+C,2,45.563128,-84.706747,5000
+C,3,45.563069,-84.705397,5000
+C,4,45.563751,-84.703382,5000
+D,1,45.560897,-84.70701,5000
+D,2,45.56203,-84.705022,5000
+D,3,45.560387,-84.704516,5000
+D,4,45.561776,-84.70364,5000

```

Comment:
Perhaps a non-issue, but this might be an unrealistic amount of precision for our coordinates. Based on [this](http://www.longitudestore.com/how-big-is-one-gps-degree.html), I calculated that coordinates to the nearest 10<sup>-6</sup> suggest a geographic precision of 5 - 10 cm (10<sup>-6</sup> degrees x 3600 sec/degree x 24 m/sec). Based on our earlier conversation, it might be more honest to drop the last 1-2 (or more?) decimals in these values.

Reply:
Also, it may be useful to distinguish these `Latitude`/`Longitude` columns from the ones in the subplots table, if for no other reason than to make it possible to do `dplyr::*_join` without the explicit `by` argument. As it is, a "natural" join of those tables will also try to join on the `Latitude` and `Longitude` columns because they have the same name.

Reply:
>unrealistic amount of precision
@atkinsjeff OK/appropriate do you think to reduce from 6 to 4 digits of precision?

>distinguish these Latitude/Longitude columns 
Ah, good point, will do.